packages/genesys-spark-components/src/components/stable/gux-popover/example.html

```diff
@@ -12,9 +17,9 @@ <h1>gux-popover</h1>
       <div class="popover-content-wrapper">
         <div>Popover Content</div>
         <div class="popover-content">
-          <gux-button accent="ghost" onclick="notify(event)">
-            Action 3
-          </gux-button>
+          <gux-button-slot accent="ghost">
+            <button type="button" autofocus>Action 3</button>
```

Comment:
Trying to use an autofocus attribute like the gux-modal does to focus an element on popover load. This doesn't seem to be working yet so I'll keep messing with it to get this element to autofocus on popover load.